pkgs/top-level/all-packages.nix

```
@@ -22076,7 +22076,10 @@ with pkgs;
 
   cpustat = callPackage ../os-specific/linux/cpustat { };
 
-  cockroachdb = callPackage ../servers/sql/cockroachdb { };
+  inherit (import ../servers/sql/cockroachdb pkgs)
```

```suggestion
  inherit (callPackage ../servers/sql/cockroachdb pkgs { })
```

See https://github.com/NixOS/nixpkgs/pull/152626/files#r776826761 for the corresponding change.